setup.py

```diff
@@ -34,16 +34,21 @@
           'idna>=2.5',
           'typing ; python_version<"3.5"',
       ],
+      python_requires='>=2.6, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
       classifiers=[
           'Topic :: Utilities',
           'Intended Audience :: Developers',
           'Topic :: Software Development :: Libraries',
           'Development Status :: 5 - Production/Stable',
+          'Programming Language :: Python :: 2',
           'Programming Language :: Python :: 2.6',
           'Programming Language :: Python :: 2.7',
+          'Programming Language :: Python :: 3',
```

Comment:
Are these two additions appropriate when we list specific sub-versions that are supported?
If yes, should we add `Programming Language :: Python`?

Reply:
Yes, it's common to list those as well. Sometimes `Programming Language :: Python` is there too.

For example, see pip:

* https://github.com/pypa/pip/blob/master/setup.py

Shall I add that one also?

Reply:
Sorry, I forgot this was still open.

It's hard to argue with the `pip` team about packaging, so following their lead seems like a good idea.